src/PublicKey/RsaOptions.php

```diff
@@ -52,6 +52,13 @@ class RsaOptions extends AbstractOptions
     protected $binaryOutput = true;
 
     /**
+     * OPENSSL padding
+     *
+     * @var int
```

Comment:
Same as below: the comment hides some meaning (which is that this variable is OpenSSL-specific)